assets/src/scripts/components/map/nldiMapping.js

```diff
@@ -97,22 +94,24 @@ export const addNldi = function(map, legendControl, siteno) {
     const upStreamFlow = nldiUrl + '/'+ featureSource + '/' + featureId + '/navigate/' + upstreamNavigation + distanceParam;
     const downStreamFlow = nldiUrl + '/'+ featureSource + '/' + featureId + '/navigate/' + downstreamNavigation + distanceParam;
 
-    const nldiLines = [
-        {url : upStreamFlow, style : upstreamLineStyle},
-        {url : downStreamFlow, style : downstreamLineStyle}
-    ];
-
-    const nldiPoints = [
-        {url : upStreamSites, style : geojsonMarkerOptions},
-        {url : downStreamSites, style : geojsonMarkerOptions}
-    ];
-
-    nldiLines.forEach(function(pair) {
-        addNldiLinesToMap(pair.url, pair.style);
-    });
-
-    nldiPoints.forEach(function(pair) {
-        addNldiPointsToMap(pair.url, pair.style);
+    const nldiUpStreamLines = {url : upStreamFlow, style : upstreamLineStyle};
```

Comment:
I don't think these constants serve any purpose. Just reference the appropriate constant in the set of *Promise constants.